charts/mobu/templates/service.yaml

```diff
@@ -3,12 +3,12 @@ kind: Service
 metadata:
   name: {{ include "mobu.fullname" . }}
   labels:
-{{ include "mobu.labels" . | indent 4 }}
+    {{- include "mobu.labels" . | nindent 4 }}
 spec:
+  type: ClusterIP
   ports:
     - port: 80
-      targetPort: 8080
-      protocol: TCP
+      targetPort: "http"
```

Comment:
I thought the container was still running on 8080 in the pod?

Reply:
This will still use port 8080, but it does so by name rather than by port number. (The name here is the name of a port specified in the deployment, not a name from `/etc/services`.) It means less repeating of the configured port number should it ever change.